packages/vuetify/src/components/VLazy/VLazy.tsx

```diff
@@ -0,0 +1,70 @@
+// Composables
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeProps, maybeTransition } from '@/util'
+import { makeTagProps } from '@/composables/tag'
+import { makeTransitionProps } from '@/composables/transition'
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Directives
+import intersect from '@/directives/intersect'
+
+// Utilities
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'VLazy',
+
+  directives: { intersect },
+
+  props: makeProps({
+    modelValue: Boolean,
+    options: {
+      type: Object,
```

Comment:
```suggestion
      type: Object as PropType<IntersectionObserverInit>,
```
(i believe it's the correct type)